astroquery/conftest.py

```diff
@@ -40,6 +40,9 @@
 # using Astropy's
 try:
     from .version import version
+    # add '_testrun' to the version name so that the user-agent indicates that
+    # it's being run in a test
+    version += '_testrun'
```

Comment:
Instead of hacking version, maybe define a new var like `IS_RUNNING_TEST`?

Reply:
this approach is very simple, though, since no other changes are required at all (version is already propagated through the user-agent).  I'm not sure there's any real motivation to make any other change, since what we're trying to communicate is a particular variant of astroquery tests being called.